common/setups/rasr/gmm_system.py

```diff
@@ -559,16 +559,16 @@ def extract_vtln_features(
         :param kwargs:
         :return:
         """
+        self.vtln_features(
+            name=name,
+            corpus=train_corpus_key,
+            raw_feature_flow=self.feature_flows[train_corpus_key][raw_feature_flow_key],
+            warping_map=self.vtln_files[train_corpus_key][
+                vtln_files_key + "_warping_map"
+            ],
+            **kwargs,
+        )
         for c in eval_corpora_keys:
-            self.vtln_features(
-                name=name,
-                corpus=train_corpus_key,
-                raw_feature_flow=self.feature_flows[c][raw_feature_flow_key],
-                warping_map=self.vtln_files[train_corpus_key][
-                    vtln_files_key + "_warping_map"
-                ],
-                **kwargs,
-            )
```

Comment:
how did this ever make sense? it will extract the training data as vtln features again and again (ok, maybe `self.feature_flows[c][raw_feature_flow_key]` is constant for all c, so the same job is created) and overwrites the `self.feature_flows[train_corpus_key][name+"+vtln"]` every time.

Reply:
I did not make sense, thus I changed it. The hashes did not change though.